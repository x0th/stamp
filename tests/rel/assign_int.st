Object count = 10;
Object.count
