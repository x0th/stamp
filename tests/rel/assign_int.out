STDOUT:
10
STDERR:
